app/features/build_runner/build_runner.rb

```diff
@@ -376,10 +380,13 @@ def save_build_status_locally!
     def save_build_status_source!
       status_context = project.project_name
 
+      build_path = FastlaneCI::BuildController.build_url(project_id: project.id, build_number: current_build.number)
+      build_url = FastlaneCI.env.ci_base_url + build_path
```

Comment:
I'm pretty sure this crashes if the host `ci_base_url` is `nil`, inside the `ci_base_url` method we could default to the local area IP address for now?